sequencer/src/options.rs

```diff
@@ -80,6 +81,9 @@ pub struct Options {
     )]
     pub webserver_poll_interval: Duration,
 
+    // Sequencer Private Staking key
+    #[clap(short, long, env = "ESPRESSO_SEQUENCER_PRIVATE_STAKING_KEY")]
+    pub private_staking_key: BLSPrivKey,
```

Comment:
I think we need to add some code to generate such keys for the docker compose demo, or have a "demo mode" where the keys are fetched from the orchestrator.

Reply:
I can make it optional and use the generated key from the orchestrator config if the arg is not provided

Reply:
I think we should add a tool like `bin/keygen` to generate keys. Then we can use that to create some random keys that we hardcode into the docker-compose file. I would feel slightly better about doing the demo that way and not having the insecure "use orchestrator keys" as a code path in the main sequencer node at all